tests/specs/tram-spec.js

```diff
@@ -73,6 +73,28 @@ const tests = (Tram) => describe('Tram', () => {
     })
   })
 
+  describe('addListener', () => {
```

Comment:
## Tests
New tests, pretty much just verify that the function was added properly. Uses internal knowledge of hover-engine, which is meh, but we do it above, so if we plan on changing it, we should do it for both.